spec/fixtures/files/peer_reviews/.gitignore

```diff
@@ -0,0 +1 @@
+temp_peer_review.csv
```

Comment:
Yeah, I think it's safe to assume no one will use the name, but maybe prefix it with an underscore just to be safe.

I'm surprised it's automatically deleted, but I'll go with it.